sensor_msgs/PointCloud2 reference_pc
---
int8 state
string message